recipe/meta.yaml

```diff
@@ -1,36 +1,51 @@
 {% set name = "pox" %}
-{% set version = "0.3.2" %}
+{% set version = "0.3.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e825225297638d6e3d49415f8cfb65407a5d15e56f2fb7fe9d9b9e3050c65ee1
+  sha256: e1ced66f2a0c92a58cf3646bc7ccb8b4773d40884b76f85eeda0670474871667
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python =2.7|>=3.6
+    - setuptools
+    - wheel
+    - python
   run:
-    - python =2.7|>=3.6
+    - python
```

Comment:
does dummy_version is it just placeholder or real package ?
https://github.com/uqfoundation/pox/blob/53a8d8875534c97d5c8b7452c42c1ea411fbf9bf/setup.py#L90C11-L90C24

Reply:
Probably it's a test, because it is commented out in
https://github.com/uqfoundation/pox/blob/53a8d8875534c97d5c8b7452c42c1ea411fbf9bf/setup.py#L97

But nice catch, I'd missed that